src/crudeditor-lib/middleware/notifications.js

```diff
@@ -145,7 +140,29 @@ const eventsMiddleware = ({ context, modelDefinition }) => store => next => acti
     default:
   }
 
-  return next(action)
+  if ([
+    CREATE_ALL_INSTANCE_FIELDS_VALIDATE,
+    EDIT_ALL_INSTANCE_FIELDS_VALIDATE
+  ].indexOf(action.type) > -1) {
+    const currentView = store.getState().common.activeViewName;
+
+    next(action);
```

Comment:
result of next(action) call must be saved and returned from the middleware. See appStateChangeDetect.js as an example.